p5js/TeachableMachine/SoundModel_TM/index.html

```diff
@@ -3,17 +3,16 @@
 <head>
   <meta charset="UTF-8">
   <title>Webcam Image Classification using a pre-trained customized model and p5.js</title>
-
   <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.9.0/p5.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.9.0/addons/p5.dom.min.js"></script>
-
   <script src="http://localhost:8080/ml5.js" type="text/javascript"></script>
 </head>
 
 <body>
-  <h1>Webcam Image Classification using a pre-trained customized model in Google's Teachable Machine and p5.js</h1>
-  <p>If you cover your webcam, this model will classify it as "nighttime", 
-      otherwise will classify anything else else "daytime" </p>
+  <h1>Teachable Machine</h1>
+  <p>This is a demonstration of sound classification using a model trained with Google's Teachable Machine project. If
+    you clap, this model will classify the sound as "clap." (Until then it the canvas displays "listening.")
```

Comment:
See typo: "until then it the canvas..."  == >`Until a clap is detected the canvas displays "listening."`

Reply:
Fixed!